sensor/common/networkflow/manager/indicator/key.go

```diff
@@ -114,24 +48,6 @@ func hashToHexString(hash uint64) string {
 	return string(buf)
 }
 
-func formatPortAndProtocol(buf *strings.Builder, port uint16, protocol storage.L4Protocol) {
-	buf.WriteByte(':')
-	buf.WriteString(strconv.FormatUint(uint64(port), 10))
-	buf.WriteByte(':')
-	buf.WriteString(strconv.FormatUint(uint64(protocol), 10))
-}
-
-// buildStringKey is a memory allocation optimized version of `buf.WriteString(strings.Join(parts..., ":"))`.
-// Benchmarks show 50% runtime and 50% memory allocation for the current impl when compared against `strings.Join`.
-func buildStringKey(buf *strings.Builder, parts ...string) {
-	for i, part := range parts {
-		if i > 0 {
-			buf.WriteByte(':')
-		}
-		buf.WriteString(part)
-	}
-}
-
 func hashStrings(h hash.Hash64, strs ...string) {
```

Comment:
How about copying @johannes94 work https://github.com/stackrox/hashstructure/commit/9f325b1e6fe99eb610951626750c44dddb3febf5

Reply:
Using unsafe.Slice for that? I would need to check if that would work here, because I understand that is would operate on the underlying data and possibly change it. I will try it out